### ISSUE TITLE
False positive RCS1228

### ISSUE THREAD

**Issue by daze99** [dotnet/roslynator#1322]
I am using the version 4.7.0

The line `/// <param name="date"></param>` is flagged.
```csharp
    /// <summary>
    /// Test Method <c>Test</c>.
    /// </summary>
    /// <param name="date"></param>
    /// <returns>date as string.</returns>
    private static string Test(DateTime date)
    {
        return date.ToString();
    }
```

**Comment by daze99** [comment]
It probably makes no sense. Let's close this issue.
